### PR TITLE
adding the argo-clinical subgraph

### PR DESCRIPTION
**Pull request by UmmulkiramR**
argo clinical will now be a part of the rdpc-gateway apollo federation service.

### CODE REVIEW

File: package_bckup.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (joneubank) [inline package_bckup.json:1]: Maybe we don't want to add your local backups? ;)

File: src/index.ts [hunk 2]
```diff
@@ -54,6 +59,9 @@ const gateway = new ApolloGateway({
   },
 });
 
+
+console.log('gateway: ' + gateway.schema);
```
Comment (joneubank) [inline src/index.ts:63]: Debug comments, can remove?

**Review by joneubank — CHANGES_REQUESTED** [review]
Looks good, just wanna remove the local dev files/logs?

File: package_bckup.json (hunk 1 above)
Comment (UmmulkiramR) [reply package_bckup.json]: File removed. Thanks for pointing out.

File: src/index.ts (hunk 2 above)
Comment (UmmulkiramR) [reply src/index.ts]: yes, removed. Thanks.

**Comment by UmmulkiramR** [comment]
closing this as https://github.com/icgc-argo/rdpc-gateway/pull/95 is approved and merged.

**Pull request closed without merging**